src/Button/GreenButton.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components'
+import Button from './Button'
+
+const GreenButton = styled(Button)`
+  background-color: ${props => props.theme.colors.green};
+
+  &:hover {
+    background-color: ${props => props.theme.colors.green1};
+  }
+`
+export default GreenButton
```

Comment:
Can this component become from <GreenButton /> to <ColorButton color='green' />? (thinking that maybe the button can be the colored button more dynamic instead of a fixed color; 
 same thought applies to the `RedButton`)

Reply:
We discussed this in person, but my suggestion was to start with explicitly named components, and abstract into props later if we need it. Part of the thinking is that having a component available is a little more obvious than props. 

Based on what I've seen in the product, we're mostly only using green and blue buttons, and by handling these as separate components we'll need less logic to handle hover, focus, and active states when those are added in.